application/src/main/java/org/thingsboard/server/service/install/update/DefaultDataUpdateService.java

```diff
@@ -716,73 +211,6 @@ boolean convertDeviceProfileAlarmRulesForVersion330(JsonNode spec) {
-                }
-
-                @Override
-                protected PageData<TenantProfile> findEntities(String id, PageLink pageLink) {
-                    return tenantProfileService.findTenantProfiles(TenantId.SYS_TENANT_ID, pageLink);
-                }
-
-                @Override
-                protected void updateEntity(TenantProfile tenantProfile) {
-                    updateTenantProfileQueueConfiguration(tenantProfile);
-                }
-            };
-
-    private void updateTenantProfileQueueConfiguration(TenantProfile profile) {
-        try {
-            List<TenantProfileQueueConfiguration> queueConfiguration = profile.getProfileData().getQueueConfiguration();
-            if (profile.isIsolatedTbRuleEngine() && (queueConfiguration == null || queueConfiguration.isEmpty())) {
-                TenantProfileQueueConfiguration mainQueueConfig = getMainQueueConfiguration();
-                profile.getProfileData().setQueueConfiguration(Collections.singletonList((mainQueueConfig)));
-                tenantProfileService.saveTenantProfile(TenantId.SYS_TENANT_ID, profile);
-                List<TenantId> isolatedTenants = tenantService.findTenantIdsByTenantProfileId(profile.getId());
-                isolatedTenants.forEach(tenantId -> {
-                    queueService.saveQueue(new Queue(tenantId, mainQueueConfig));
-                });
-            }
-        } catch (Exception e) {
-            log.error("Failed to update tenant profile queue configuration name=[" + profile.getName() + "], id=[" + profile.getId().getId() + "]", e);
-        }
-    }
-
-    private TenantProfileQueueConfiguration getMainQueueConfiguration() {
-        TenantProfileQueueConfiguration mainQueueConfiguration = new TenantProfileQueueConfiguration();
-        mainQueueConfiguration.setName(DataConstants.MAIN_QUEUE_NAME);
-        mainQueueConfiguration.setTopic(DataConstants.MAIN_QUEUE_TOPIC);
-        mainQueueConfiguration.setPollInterval(25);
-        mainQueueConfiguration.setPartitions(10);
-        mainQueueConfiguration.setConsumerPerPartition(true);
-        mainQueueConfiguration.setPackProcessingTimeout(2000);
-        SubmitStrategy mainQueueSubmitStrategy = new SubmitStrategy();
-        mainQueueSubmitStrategy.setType(SubmitStrategyType.BURST);
-        mainQueueSubmitStrategy.setBatchSize(1000);
-        mainQueueConfiguration.setSubmitStrategy(mainQueueSubmitStrategy);
-        ProcessingStrategy mainQueueProcessingStrategy = new ProcessingStrategy();
-        mainQueueProcessingStrategy.setType(ProcessingStrategyType.SKIP_ALL_FAILURES);
-        mainQueueProcessingStrategy.setRetries(3);
-        mainQueueProcessingStrategy.setFailurePercentage(0);
-        mainQueueProcessingStrategy.setPauseBetweenRetries(3);
-        mainQueueProcessingStrategy.setMaxPauseBetweenRetries(3);
-        mainQueueConfiguration.setProcessingStrategy(mainQueueProcessingStrategy);
-        return mainQueueConfiguration;
-    }
```

Comment:
Let's clean up yml files as well (remove `queue.rule-engine.queues` configurations, they are not used anymore)